usrsctplib/usrsctp.h

```diff
@@ -1030,6 +1030,9 @@ usrsctp_deregister_address(void *);
 int
 usrsctp_set_ulpinfo(struct socket *, void *);
 
+int
+usrsctp_set_sb_threshold(struct socket *, uint32_t sb_threshold);
```

Comment:
Please remove `sb_threshold` for consistency.

Reply:
Done.

Reply:
Really?

Reply:
Right sorry, merged with the upstream.